roborock/devices/local_channel.py

```diff
@@ -45,12 +57,77 @@ def __init__(self, host: str, local_key: str):
         self._protocol: _LocalProtocol | None = None
         self._subscribers: CallbackList[RoborockMessage] = CallbackList(_LOGGER)
         self._is_connected = False
-
-        self._decoder: Decoder = create_local_decoder(local_key)
-        self._encoder: Encoder = create_local_encoder(local_key)
+        self._local_key = local_key
```

Comment:
I think we can drop `_local_key` and `_connect_nonce` too? Either `set _params` here or pass to `_update_encoder_decoder` explcitly